src/go/rpk/pkg/cli/cmd/wasm/remove.go

```diff
@@ -36,12 +37,20 @@ func NewRemoveCommand(
 			}
 			return remove(
 				name,
+				coproc_type,
 				producer,
 				admin,
 			)
 		},
 	}
 
+	command.Flags().StringVar(
```

Comment:
same issue here, maybe there can be some shared code that lives in wasm/common.go

Reply:
Done